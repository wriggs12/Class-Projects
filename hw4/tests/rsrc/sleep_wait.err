[00000.000000] STARTUP
[00000.000024] PROMPT
[00000.100255] INPUT run echo hello world
[00000.100487] CHANGE 2186529: none -> running
[00000.101289] SIGNAL 17
[00000.101321] CHANGE 2186529: running -> stopped
[00000.101374] PROMPT
[00000.201494] INPUT run sleep 1
[00000.201693] CHANGE 2186530: none -> running
[00000.202350] SIGNAL 17
[00000.202375] CHANGE 2186530: running -> stopped
[00000.202413] PROMPT
[00000.302552] INPUT cont 1
[00000.302599] CHANGE 2186530: stopped -> running
[00000.302630] PROMPT
[00000.402743] INPUT cont 0
[00000.402769] CHANGE 2186529: stopped -> running
[00000.402786] PROMPT
hello world
[00000.403547] SIGNAL 17
[00000.403570] CHANGE 2186529: running -> dead
[00000.503043] INPUT wait 1
[00001.304229] SIGNAL 17
[00001.304296] CHANGE 2186530: running -> dead
[00001.304345] PROMPT
[00001.404471] INPUT quit
[00001.404505] SHUTDOWN
