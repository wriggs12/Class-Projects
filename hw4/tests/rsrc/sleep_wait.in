run echo hello world
run sleep 1
cont 1
cont 0
wait 1
quit
