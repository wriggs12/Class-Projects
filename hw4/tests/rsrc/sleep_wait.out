0	2186529	T	running		echo hello world
0	2186529	T	stopped		echo hello world
1	2186530	T	running		sleep 1
1	2186530	T	stopped		sleep 1
1	2186530	T	running		sleep 1
0	2186529	T	running		echo hello world
0	2186529	T	dead	0x0	echo hello world
1	2186530	T	dead	0x0	sleep 1
